.babelrc

```diff
@@ -2,26 +2,26 @@
   "presets": [
     "react",
     "flow",
-    ["env", {
-      "targets": {
-        "browsers": [
-          "ie >= 11",
-          "last 1 Edge version",
-          "last 1 Firefox version",
-          "last 1 Chrome version",
-          "last 1 Safari version",
-          "last 1 iOS version",
-          "last 1 Android version",
-          "last 1 ChromeAndroid version"
-        ]
-      }
-    }]
+    ["env", { "modules": false }]
   ],
   "plugins": [
-    "transform-runtime",
     "transform-class-properties",
     "transform-object-rest-spread",
     "transform-export-extensions"
   ],
-  "comments": false
-}
\ No newline at end of file
+  "comments": false,
+  "env": {
+    "cjs": {
+      "plugins": [
+        ["transform-runtime", { "polyfill": false }],
```

Comment:
Should we be doing this?

Reply:
runtime adds this polyfills
- babel-runtime/core-js/object/assign
- babel-runtime/core-js/object/get-prototype-of
- babel-runtime/core-js/object/keys
the case is that rollup do not adds any polyfill.

Reply:
Won't this affect consumers who are using the cjs build though?

Reply:
Depends on browser support. I just get rid from get-prototype-of polyfill with loose mode.
Assign can be removed with spread operator which reuses extend helpers which is already in bundle. Not sure about `Object.keys`, but it's even es5.

Reply:
I see, it's only one place with mutating. Maybe local polyfill?

Reply:
We'll have to check on ie11. I think it safest to leave the polyfills in for now given they are so small

Reply:
Thoughts @TrySound ?

Reply:
I updated stats. It seems like in polyfills has bigger impact external-helpers and loose mode. Not sure. I just added `object-assign` polyfill, which is small and the same as babel adds.

Reply:
I think for the sake of 3kb it is worth keeping the babel polyfills for simplicity and for safety

Reply:
112.0K vs 107.8K (4.2kb)

Reply:
I like the idea of keeping the babel one also as we go forward as it can add things we use dynamically

Reply:
@lukebatchelor ^

Reply:
The problem is that babel is not actually good with helpers. It can detect only static stuff like `Object.keys`, but cannot `[].reduce`. So you still need to check by yourself. With current configuration runtime adds only `Object.keys` polyfill, which is useless.

Reply:
Hmmm... I do agree it's kind dvof useless, but my worry would be someone adding in a usage later which would then break ie11.

There really aren't many things it helps with, but might be worth keeping just for the safety? Unless there's a simple eslint rule we can turn on to avoid it?

Reply:
It would make me feel better using the babel auto polyfill incase we lean on it in the future. I know that what you have is strictly smaller - but i think the safety of using babel is worth keeping it for now.

If you can update the PR @TrySound to include that change we should be good to go!

Thanks for your perseverance with this one @TrySound !